src/modules/Header/components/Header.js

```diff
@@ -26,23 +30,47 @@ const HeaderBarUpper = styled.div`
   width: 100%;
   background: ${COLORS.WHITE};
   max-height: 50px;
-
 `
 
 const HeaderBarLower = styled.div`
   display: flex;
   justify-content: ${props => props.justifyContent ? props.justifyContent : 'flex-start'};
   margin: 0 auto;
-  padding-left:20px;
   width: 100%;
   background: ${COLORS.OIVA_MENU_BG_COLOR};
   max-height: 50px;
+  overflow: hidden;
+  @media ${MEDIA_QUERIES.MOBILE} {
```

Comment:
Tailwind CSS provides media queries out of the box. There shouldn't be need to define media queries like this.